### PR TITLE
Practice Pull Request

### PR DESCRIPTION
**Pull request by rachaelbradley**
Do Not Merge this Pull Request

This is for practice only. Use this to practice in line comments and suggestions.

### CODE REVIEW

File: guidelines/index.html [hunk 1]
```diff
@@ -53,6 +53,34 @@ <h3>About WCAG 3</h3>
...
 				<p>There are many differences between WCAG 2 and WCAG 3. Content that conforms to WCAG 2.2 A and AA is expected to meet most of the minimum conformance level of this new standard but, since WCAG 3 includes additional tests and different scoring mechanics, additional work will be needed to reach full conformance. Since the new standard will use a different conformance model, the Accessibility Guidelines Working Group expects that some organizations may wish to continue using WCAG 2, while others may wish to migrate to the new standard. For those that wish to migrate to the new standard, the Working Group will provide transition support materials, which may use mapping and other approaches to facilitate migration.</p>
 			</section>
+			<section>
+				<h3>Practice section</h3>
+				<p>This is a practice section. It is available to try out commenting and making suggestions on PRs. A poem is below to practice on.</p>
```
Comment (rachaelbradley) [inline guidelines/index.html:58]: This is a comment.

File: guidelines/index.html [hunk 2]
```diff
@@ -53,6 +53,34 @@ <h3>About WCAG 3</h3>
...
+
+
+					<h4>A Jelly-Fish</h4>
+					<b>Marianne Moore 1887 – 1972</b>
+					<p>Visible, invisible,</p>
```
Comment (rachaelbradley) [inline guidelines/index.html:63]:
```suggestion
					<p>Visible, invisible, or not</p>
```

File: guidelines/index.html (hunk 1 above)
Comment (priddy26) [reply guidelines/index.html:58]: test

File: guidelines/index.html [hunk 3]
```diff
@@ -53,6 +53,34 @@ <h3>About WCAG 3</h3>
...
+					<p>An amber-colored amethyst</p>
+					<p>Inhabits it; your arm</p>
+					<p>Approaches, and</p>
+					<p>It opens and</p>
+					<p>It closes;</p>
+					<p>You have meant</p>
+					<p>To catch it,</p>
+					<p>And it shrivels;</p>
+					<p>You abandon</p>
+					<p>Your intent—</p>
```
Comment (JediLin) [inline guidelines/index.html:74]: a comment to line 74

File: guidelines/index.html (hunk 2 above)
Comment (priddy26) [reply guidelines/index.html:63]:
```suggestion
					<p>Visible, invisible, test</p>
```

File: guidelines/index.html (hunk 3 above)
Comment (JediLin) [inline guidelines/index.html:74]:
```suggestion
					<p>Your and my intent—</p>
```

File: guidelines/index.html [hunk 4]
```diff
@@ -53,6 +53,34 @@ <h3>About WCAG 3</h3>
...
+					<p>Visible, invisible,</p>
+					<p>A fluctuating charm,</p>
+					<p>An amber-colored amethyst</p>
+					<p>Inhabits it; your arm</p>
+					<p>Approaches, and</p>
+					<p>It opens and</p>
+					<p>It closes;</p>
+					<p>You have meant</p>
+					<p>To catch it,</p>
+					<p>And it shrivels;</p>
```
Comment (gradualclearing) [inline guidelines/index.html:72]: Test comment
```suggestion
					<p>And it shrivels</p>
```

File: guidelines/index.html [hunk 5]
```diff
@@ -53,6 +53,34 @@ <h3>About WCAG 3</h3>
...
+
+
+					<h4>A Jelly-Fish</h4>
+					<b>Marianne Moore 1887 – 1972</b>
+					<p>Visible, invisible,</p>
+					<p>A fluctuating charm,</p>
+					<p>An amber-colored amethyst</p>
+					<p>Inhabits it; your arm</p>
+					<p>Approaches, and</p>
+					<p>It opens and</p>
```
Comment (MakotoUeki) [inline guidelines/index.html:68]:
```suggestion
					<p>It opens and/or</p>
```

File: guidelines/index.html [hunk 6]
```diff
@@ -53,6 +53,34 @@ <h3>About WCAG 3</h3>
...
+					<p>It closes;</p>
+					<p>You have meant</p>
+					<p>To catch it,</p>
+					<p>And it shrivels;</p>
+					<p>You abandon</p>
+					<p>Your intent—</p>
+					<p>It opens, and it</p>
+					<p>Closes and you</p>
+					<p>Reach for it—</p>
+					<p>The blue</p>
```
Comment (MakotoUeki) [inline guidelines/index.html:78]: Why blue?

**Pull request closed without merging**